docs/samples/python/Presidio_with_Flair_as_a_3rd_party_recognizer

```diff
@@ -1,46 +1,72 @@
-# Integrating external Flair model with Presidio
-# This is a basic way to include Flair as a 3rd party recognizer for Presidio-Analyzer.
+import logging
+from typing import Optional, List, Tuple, Set
+
+from presidio_analyzer import (
+    RecognizerResult,
+    EntityRecognizer,
+    AnalysisExplanation,
+    AnalyzerEngine,
+)
 
-from presidio_analyzer import AnalysisExplanation, EntityRecognizer, RecognizerResult, AnalyzerEngine
 from presidio_analyzer.nlp_engine import NlpArtifacts
 
 from flair.data import Sentence
 from flair.models import SequenceTagger
 
-from typing import List
+
+logger = logging.getLogger("presidio-analyzer")
+
 
 class FlairRecognizer(EntityRecognizer):
+
+    ENTITIES = ["ORGANIZATION", "MISCELLANEOUS", "LOCATION", "PERSON",
+                ]
+
+    DEFAULT_EXPLANATION = "Identified as {} by Flair's Named Entity Recognition"
+
+    CHECK_LABEL_GROUPS = [
+        ({"LOCATION"}, {"LOC"}),
+        ({"PERSON"}, {"PER"}),
+        ({"MISCELLANEOUS"}, {"MISC"}),
```

Comment:
What goes under MISC? is it PII? if not, I think it would be best to ignore it.

Reply:
MISC is a box to put other name not classified as PER, LOC or ORG.
In the text (Antoni Gaudi was the first architect of Sagrada Familia, a Catholic cathedral in Barcelona.)  it goes to Catholic. I leaved because it was something similar to Presidio's NRP, but it includes more things, like names of diseases for example.
I think there are two options, the first one is to ignore it. May be is the best. The second one is to assimilate to NRP, but it could be confusing when the detections will not be related to nationality, religious or politics.
I'm good to apply what you prefer.

Reply:
Actually thinking about it, it makes sense to have A Misc category in general, for cases like this. Let's keep it as it then. Thanks for the clarification